wizard/src/utils/country_mappings.csv

```diff
@@ -0,0 +1,272 @@
+id,name,iso_code,country_alpha_3_code,country_numeric_code
```

Comment:
In case it is easier to process, we could have a json file, generated as below.

`echo "SELECT id,name,iso_code,country_alpha_3_code,country_numeric_code FROM \`datcom-store.dc_kg_latest.Place\` WHERE type = 'Country'" | bq query --format=prettyjson --use_legacy_sql=false`

Might be useful to add regeneration step to the README.

Reply:
thanks!